docs/source/markdown/podman-search.1.md

```diff
@@ -140,6 +147,36 @@ fedoraproject.org   registry.fedoraproject.org/f25/kubernetes-proxy
 fedoraproject.org   registry.fedoraproject.org/f25/kubernetes-scheduler                          0
 fedoraproject.org   registry.fedoraproject.org/f25/mariadb                                       0
 ```
+
+```
+$ podman search --list-tags  registry.redhat.io/rhel
+NAME                      TAG
+registry.redhat.io/rhel   7.3-74
+registry.redhat.io/rhel   7.6-301
+registry.redhat.io/rhel   7.1-9
+registry.redhat.io/rhel   7.3-89
+registry.redhat.io/rhel   7.3-66
+registry.redhat.io/rhel   7.5-424
+registry.redhat.io/rhel   7.5-245.1527091554
+registry.redhat.io/rhel   7.3-82
+registry.redhat.io/rhel   7.4-129
+registry.redhat.io/rhel   7.4-120
+registry.redhat.io/rhel   7.6-122
+registry.redhat.io/rhel   7.7-481
+registry.redhat.io/rhel   7.2
+registry.redhat.io/rhel   7.5-409
+registry.redhat.io/rhel   7.2-75
+registry.redhat.io/rhel   7.8-356
+registry.redhat.io/rhel   7.7-348
+registry.redhat.io/rhel   7.7
+registry.redhat.io/rhel   7.4
+registry.redhat.io/rhel   7.5
+registry.redhat.io/rhel   7.5-409.1533127727
+registry.redhat.io/rhel   7.8
+registry.redhat.io/rhel   7.5-404
+registry.redhat.io/rhel   7.4-105
+registry.redhat.io/rhel   7.2-35
+```
```

Comment:
Although this shows what gets returned, for the sake of brevity, I think I'd just show 3 or 4 lines and then a new line with `...` or `(multiple lines removed for brevity)`

Reply:
And I'm going back/forth on this.  Do others have thoughts?

Reply:
Yes let's list 3 and then ...